release-notes/opendistro-for-elasticsearch.job-scheduler.release-notes-1.10.0.0.md

```diff
@@ -0,0 +1,4 @@
+## 2020-08-18, version 1.10.0.0
+
+### New Features
```

Comment:
Peter created a standard guideline for release note. pls have a check.  I remember we don't use "New Features" for upstream upgrade.

Reply:
Thank you! I will apply the automated script for the release notes asap.

Reply:
I have modified the format, and I will address the format of past release notes in another PR.